src/karmen_backend/server/routes/gcodes.py

```diff
@@ -45,7 +45,7 @@ def make_gcode_response(gcode, fields=None, user_mapping=None):
     return response
 
 
-@app.route("/organizations/<org_uuid>/gcodes", methods=["GET"])
+# @app.route("/organizations/<org_uuid>/gcodes", methods=["GET"])
```

Comment:
Please remove commented-out and unused code (applies to multiple places).

Reply:
`@app.route` could go, but I would keep at least comment with URL and method, so you don't have to check the specs every time you are looking for something.

Reply:
Agreed.